src/web/mod.rs

```diff
@@ -864,11 +847,6 @@ mod test {
         });
     }
 
-    #[test]
-    fn test_templates_are_valid() {
-        handlebars_engine().expect("Failed to load handlebar templates");
-    }
-
```

Comment:
Please add an equivalent of this test for tera.

Reply:
There already [is one](https://github.com/rust-lang/docs.rs/blob/master/src/web/page/templates.rs#L257-L265)